src/components/_navbar.scss

```diff
@@ -312,16 +312,16 @@
   display: flex;
   flex-direction: row !important;
   align-items: center;
-  color: $fluid-navbar-brand !important;
-  background-color: $fluid-navbar-brand-bg;
+  color: $navbar-brand-color !important;
+  background-color: $navbar-brand-bg;
   text-transform: uppercase;
   white-space: nowrap;
   line-height: px(60);
   overflow: hidden;
   @include transition($transition-collapse-all);
 
   &[data-toggle="collapse"] {
```

Comment:
Tight coupling this theme to a specific JS framework is bad practice.

Reply:
We're explicitly tight-coupling this to Bootstrap and this is part of Bootstrap. The other way would be to add a custom class and that would break with our design goals.

Reply:
Yes, I am referring to the data-toggle="collapse" thing and no, you're not tight coupling to Bootstrap, you're tight coupling to jQuery which is not in the specs for Fluid. I can also say with 100% certainty  that there is nothing in Bootstrap with a "data-*" type selector (I've searched). Thus, either by adding a class (preferred) or by adding a "data-*" attribute selector you should explicitly state this in the documentation for this component.

Reply:
Also sorry for referencing old PR code but I only now have noticed this :disappointed:

Reply:
But 1) Bootstrap is tight-coupled to jQuery and 2) look at the docs on Collapse here: https://getbootstrap.com/docs/4.1/components/collapse/ -- it uses data-toggle="collapse", even explicitly stating that "...the data-toggle="collapse" is required."

Reply:
And again, if this is your only problem with this PR, then please let's discuss it under a separate issue and not hold up this PR. I have a ton of work waiting after this.